pkg-manager/plugin-commands-installation/test/update/interactive.ts

```diff
@@ -354,3 +354,78 @@ test('interactively update should ignore dependencies from the ignoreDependencie
     expect(lockfile.packages['/is-positive@2.0.0']).toBeTruthy()
   }
 })
+
+test('interactively update should update corepack config', async () => {
+  prepare({
+    name: 'project-1',
+    packageManager: 'pnpm@8.7.0',
+  })
+
+  const storeDir = path.resolve('pnpm-store')
+
+  const headerChoice = {
+    name: 'Package                                                    Current   Target            URL ',
+    disabled: true,
+    hint: '',
+    value: '',
+  }
+
+  await add.handler(
+    {
+      ...DEFAULT_OPTIONS,
+      cacheDir: path.resolve('cache'),
+      dir: process.cwd(),
+      linkWorkspacePackages: true,
+      save: false,
+      storeDir,
+    },
+    ['pnpm@8.9.2']
+  )
+
+  prompt.mockResolvedValue({
+    updateDependencies: [
+      {
+        value: 'pnpm',
+        name: chalk`pnpm 8.7.0 ❯ 8.9.2 https://pnpm.io/ `,
+      },
+    ],
+  })
+
+  prompt.mockClear()
+  // t.comment('update to compatible versions')
+  await update.handler({
+    ...DEFAULT_OPTIONS,
+    cacheDir: path.resolve('cache'),
+    dir: process.cwd(),
+    interactive: true,
+    linkWorkspacePackages: true,
+    storeDir,
+  })
+
+  expect(prompt.mock.calls[0][0].choices).toStrictEqual([
+    {
+      choices: [
+        headerChoice,
+        {
+          message: chalk`pnpm                                                         8.7.0 ❯ 8.9.2                 `,
```

Comment:
How can i get the latest pnpm version here at my test case... otherwise we will need to update the test case here everytime pnpm publish latest version. @zkochan

Reply:
could we just remove the pnpm version before checking the snapshot?
Or mock the pnpm registry endpoint with nock?

Reply:
ok, i will update here~

Reply:
done